plugins/plugin-java-debugger/che-plugin-java-debugger-ide/src/main/java/org/eclipse/che/plugin/jdb/ide/fqn/JavaFqnResolver.java

```diff
@@ -75,7 +75,7 @@ public String resolveFqn(@NotNull final VirtualFile file) {
         String projectBuilder = getProjectBuilder(projectNode);
         Map<String, List<String>> attributes = projectNode.getProjectConfig().getAttributes();
 
-        sourceFolders.addAll(attributes.containsKey(projectBuilder + ".source.folder") ? attributes.get(projectBuilder + ".source.folder")
+        sourceFolders.addAll(attributes.containsKey("java.source.folder") ? attributes.get("java.source.folder")
```

Comment:
you can use special constant which contains this attribute's name
`org.eclipse.che.ide.ext.java.shared.Constants#SOURCE_FOLDER`